pkg/service/service.go

```diff
@@ -14,6 +14,45 @@ type Health struct {
 func (s *Health) Push(context.Context, *proto.PushRequest) (*proto.PushResponse, error) {
 	return nil, errors.New("not implemented")
 }
-func (s *Health) Register(context.Context, *proto.RegisterRequest) (*proto.RegisterResponse, error) {
-	return nil, errors.New("not implemented")
+func (s Health) Register(c context.Context, reqproto.RegisterRequest) (proto.RegisterResponse, error) {
+    client, err := mongo.NewClient(options.Client().ApplyURI(mongoURI))
+    if err != nil {
+        log.Fatal().Err(err)
+    }
+    ctx, _ := context.WithTimeout(context.Background(), 10time.Second)
```

Comment:
invalid line
you have to configure your ide otherwise you will have issues like this all the time